crates/bevy_pbr/src/render/mesh_types.wgsl

```diff
@@ -28,6 +28,6 @@ struct MorphWeights {
 };
 #endif
 
-const MESH_FLAGS_SHADOW_RECEIVER_BIT: u32 = 1u;
+const MESH_FLAGS_SHADOW_RECEIVER_BIT: u32            = 1u;
```

Comment:
This extra space seems like a typo?

Reply:
I had added some flags in an iteration, and subsequently removed them but left the spaces as it aligns with the = below. I can remove it if you like.

Reply:
Removed.